web/user/src/store/live.ts

```diff
@@ -0,0 +1,32 @@
+import { defineStore } from 'pinia'
+import type { ArchiveSchedulesResponse } from '~/types/api'
+
+export const useLiveStore = defineStore('live', {
+  state: () => {
+    return {
+      archivesFetchState: {
+        isLoading: false,
+      },
+      archiveResponse: {} as ArchiveSchedulesResponse
+    }
+  },
+
+  actions: {
+    async fetchArchves(limit = 20, offset = 0): Promise<void> {
```

Comment:
AIちゃんに指摘されてるtypoだけ修正必要かも👀

```suggestion
    async fetchArchives(limit = 20, offset = 0): Promise<void> {
```

Reply:
あとは`try~catch`入れるなら`return this.errorHandler(error)`みたいにしてもらえれば🙆